apps/nsqd/nsqd.go

```diff
@@ -92,6 +92,10 @@ func nsqFlagset() *flag.FlagSet {
 	lookupdTCPAddrs := util.StringArray{}
 	flagSet.Var(&lookupdTCPAddrs, "lookupd-tcp-address", "lookupd TCP address (may be given multiple times)")
 
+	flagSet.String("gossip-address", "", "<addr>:<port> to listen on for gossip. reasonable default: 0.0.0.0:7946")
+	seedNodeAddrs := util.StringArray{}
+	flagSet.Var(&seedNodeAddrs, "seed-node-address", "TCP address of an nsqd serving as a seed node to bootstrap gossip protocol (may be given multiple times)")
```

Comment:
Naming: `gossip-seed-address`? It would be nice for this to sort with the flag above